style.css

```diff
@@ -0,0 +1,97 @@
+body{
+    width: 90%;
+    height: 350px;
+    background-color: hsl(185, 75%, 39%);
+    background-image:url(bg-pattern-top.svg), url(bg-pattern-bottom.svg);
+    background-size: 450px, 450px;
+    background-repeat: no-repeat, no-repeat;
+    background-position: 50% 50%, 50% 50% ;    
+       
+    font-family: 'Kumbh Sans', sans-serif;
+    font-size: 18px;
+    
+    
+   
+}
+/*left -50px top -100px, right -50px bottom -400px;*/ 
+.container{
+    width: 250px;
+    height: 280px;
+    background-color: aliceblue;
+    border-radius: 8px;
+    margin-left: 475px;
+    margin-top: 180px;
+    
+}
+
+
+.a1 img{
```

Comment:
Dobra praksa je da se ubaci klasa na img tag umesto da se selektuje preko dva selektora (.a img) zbog jačine selektora. Što veći broj selektora to je selektor jači